DllImportGenerator/DllImportGenerator/Marshalling/CustomNativeTypeMarshaler.cs

```diff
@@ -0,0 +1,272 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
+
+#nullable enable
```

Comment:
Is this still needed? I thought we had it enabled for all of the `DllImportGenerator` project.